install_config/configuring_authentication.adoc

```diff
@@ -200,6 +200,49 @@ identity providers are configured that identify the same set of users and map to
 the same user names.
 |===
 
+[[LookupMappingMethod]]
+==== Lookup Mapping Method
+
+When using the `lookup` mapping method, user provisioning is done by an external system, via the API.
+Typically, identities are automatically mapped to users during login. The 'lookup' mapping method automatically
+disables this automatic mapping, which requires you to map an identity and a user manually.
+
+For more information on identity objects, see the xref:../architecture/additional_concepts/other_api_objects.adoc#identity[Identity]
+user API obejct.
+
+If you are using an identity provider using the `lookup` mapping method, use the following steps for each user after configuring
+the identity provider, as shown below:
+
+. Create an {product-title} User:
++
+----
+$ oc create user <username>
+----
+
+. Create an {product-title} Identity with the name of the identity provider and 
+the name that uniquely represents this identity in the scope of the identity provider:
++
+----
+$ oc create identity <identity-provider>:<user-id-from-identity-provider>
+----
++
+The `<identity-provider>` is the name of the identity provider in the master configuration,
+as shown in the appropriate section below. 
+
+. Create a user/identity mapping for the created user and identity: 
++
+----
+$ oc create useridentitymapping <identity-provider>:<user-id-from-identity-provider> <username>
+----
+
+For example, the following commands create a {product-title} User `timw` and an Identity with identity provider `example_ldap` and the identity provider user name `timwilliams`. Then, maps this identity to the user:
+
+----
+$ oc create user timw
+$ oc create identity example_ldap:timwilliams
+$ oc create useridentitymapping example_ldap:timwilliams timw
+----
```

Comment:
I'd say that if examples are needed, they should be paired with the appropriate steps above. But at the minimum, I'd change the examples to something other than Tim's name. We use generic names like "bob" or 'alice" in places. Maybe for the providers it could be "bobprovider" or "aliceidentityprovider" as well.